docs/2_step_by_step/2_7_get_tuned_model.md

```diff
@@ -1 +1,43 @@
-# Retrieve tuned model
\ No newline at end of file
+# Retrieve tuned model
+
+Perfect!
+Now you have started the fine-tuning job in the cloud.
+However, fine-tuning takes time,
+it highly depends on the size of your training data, evaluation data and other hyper-parameters.
+
+Once fine-tuning is completed,
+you can get the tuned model with:
```

Comment:
```suggestion
However, fine-tuning takes time. It highly depends on the size of your training data, evaluation data and other hyper-parameters. Because of this, you might have to close the session and reconnect to it several times.

In the following example, we show how to connect to an existing run and download a tuned model:```

Reply:
to give some context about reconnecting..